crm_project_task/wizards/crm_create_task.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class CrmCreateTAsk(models.TransientModel):
+    _name = "crm.create.task"
+    _description = "Wizard to create task"
+
+    lead_id = fields.Many2one("crm.lead")
+    name_task = fields.Char()
```

Comment:
This is more natural in English:

```suggestion
    task_name = fields.Char()
```

Reply:
Yes, you are right